### PR TITLE
Contact me pair programming

### PR DESCRIPTION
**Pull request by alexisuwimana**
Add a contact form on the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -200,5 +200,33 @@ <h4 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" al
...
+                    <li><a href="#"><i class="fab fa-github"><img class="social-github" src="images/social/github-v.png" alt="github"></i></a></li>
+                    <li><a href="#"><i class="fab fa-linkedin-in"><img class="social-github" src="images/social/Linkedin.png" alt="github"></i></a></li>
+                    <li><a href="#"><i class="fab fa-angellist"><img class="social-thumb" src="images/social/Thumb-Enabled.png" alt="thumb"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"><img class="social-twitter" src="images/social/Twitter.png" alt="twitter"></i></a></li>
+                    <li><a href="#"><i class="fab fa-medium-m"><img class="social-M" src="images/social/M-Disabled.png" alt="M"></i></a></li>
+                </ul>
+            </div>
+     </div>
+    
+    </section>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Kindly know that you are implored to stick to the design as much as possible (e.g., font, colours, images, tests, margins) using the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).👀😊
- FIGMA DESIGN
screenshot (8)
- YOUR DESIGN
screenshot (9)

Comment (rotimiazeez) [reply index.html]: - [OPTIONAL] Kindly place your hamburger menu on the right side of your page instead of the middle to improve the UI(User Interface).😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @alexisuwimana😊, 

### Good job so far!❤💯
- Good clean code.✔
- Good commit messages. ✔

There are some issues that you still need to work on to go to the next project but you are almost there!🚀🎉

### Just a few changes and you are Unstoppable 🚅🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (fernandorpm) [reply index.html]: - [ ] **[REQUIRED CHANGES]** Hello Alexis! I can see that you've been trying to implement the requested changes, but as of right now, they're still fairly different 😞 .

What I would suggest you is the following:
```css
input {
  border: 0;
  border-bottom: 2px solid lightgray;
}

textarea {
  border: 0;
  background-color: whitesmoke;
}
```

**Review by fernandorpm — CHANGES_REQUESTED** [review]
# :warning: Changes Required on this Pull Request! :warning:
Hello there!

You are almost there! Great work implementing the requested changes, now there's just some tiny :pinching_hand: adjustments that I would like you to change for us!



- You have done a wonderful job on this project! To highlight:
  - you have demonstrated great skills in following the template :heavy_check_mark: 
  - you have paid attention to almost all of the details :heavy_check_mark: 
  - you have written a nice and clean code :heavy_check_mark: 
  - your PR is on point :ok_hand: :heavy_check_mark: 

## Required
> These changes HAVE TO be completed in order for your project to be approved. This means that the project doesn't attend to Microverse's Standard yet. I know that it can be frustrating, but they should be seen as part of your path to become an even better Developer!

- Check the comments under review marked with the tag **[REQUIRED CHANGES]**.

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!
___

## Optional
> These changes are NOT mandatory. But if you want to sky-rocket :rocket: your skills and your projects, go for it! :sunglasses:

- Check the comments below marked with the tag **[OPTIONAL]**.

___
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

File: index.html [hunk 2]
```diff
@@ -200,5 +200,35 @@ <h4 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" al
...
+<section>
+    <div class="contact-section">
+        <div class="message-box">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div class="form-container">
+            <form action="https://formspree.io/f/mknkennj" method="POST">
+                <input type="text" name="user_name" id="name" maxlength="30" placeholder="Enter your name" required>
+                <input type="email" name="Email" id="email" placeholder="Enter your email" required>
+                <textarea name="contact" id="text-msg" placeholder="Enter text here" maxlength="500" required></textarea>
```
Comment (rotimiazeez) [inline index.html:215]: - [OPTIONAL] kindly add rows and cols to increase the `Textarea` to match the Figma template design.✌

File: index.html [hunk 3]
```diff
@@ -200,5 +200,35 @@ <h4 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" al
...
+                <button id="btn-submit" type="button">Get in touch</button>
+            </form>
+        </div>
+            <div class="contact-social">
+                <ul class="contact-social-ul">
+                    <li><a href="#"><i class="fab fa-github"><img class="social-github" src="images/social/github-v.png" alt="github"></i></a></li>
+                    <li><a href="#"><i class="fab fa-linkedin-in"><img class="social-github" src="images/social/Linkedin.png" alt="github"></i></a></li>
+                    <li><a href="#"><i class="fab fa-angellist"><img class="social-thumb" src="images/social/Thumb-Enabled.png" alt="thumb"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"><img class="social-twitter" src="images/social/Twitter.png" alt="twitter"></i></a></li>
+                    <li><a href="#"><i class="fab fa-medium-m"><img class="social-M" src="images/social/M-Disabled.png" alt="M"></i></a></li>
```
Comment (rotimiazeez) [inline index.html:225]: - [OPTIONAL] Please make sure the icon colour matches the one on your preferred design😉😊.

**Review by rotimiazeez — APPROVED** [review]
# Hi team ✌, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Kindly implement the optional changes below to improve your User-Interface😉😎.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**